snippets/visualbasic/VS_Snippets_Misc/tpl_parallel/vb/simpleforeach.vb

```diff
@@ -1,33 +1,31 @@
 ﻿'<snippet03>
 ' How to: Write a Simple Parallel.ForEach Loop
 ' IMPORTANT!!!: Add reference to System.Drawing.dll
-Imports System.Threading
```

Comment:
This and the next using statement shouldn't be deleted; without them, the example won't compile.